versioned_docs/version-0.x.0/kubeslice-cli/installing-kubeslice.mdx

```diff
@@ -0,0 +1,234 @@
+    workers:
+      - name: <new-worker-cluster-name>
+        context_name: <new-worker-cluster-context>
+  kubeslice_configuration:
+    project_name: <project-namespace>
+```
+The following is an example topology file for registering a new worker cluster.
+```
+configuration:
+  cluster_configuration:
+    kube_config_path: <PATH-TO-KUBECONFIG>
+    controller:
+      name: controller
+      context_name: kind-controller
+    workers:
+      - name: worker-3
+        context_name: ks-w-3      
+  kubeslice_configuration:
+    project_name: demo
+  ```
+
+:::note
+If the KubeSlice Controller is already installed, the `-s controller` option skips installing it. The `-s kind` option skips creating a new kind 
+clusters. The `-s enterprise` option skips using enterprise installation, while the `-s demo` option skips creating a demo set up.
+:::
+
+In a demo setup, use the following command to register a new worker cluster with the KubeSlice Controller:
+```
+kubeslice-cli install --config=<new-worker-topology-yaml> -s kind -s controller -s enterprise -s demo
+```
+In an existing multi-cluster configuration, use the following command to register a new worker cluster with the KubeSlice Controller:
+```
+kubeslice-cli install --config-<new-worker-topology-yaml> -s controller
+```
+
+## Onboarding Namespaces
+
+To onboard your existing namespaces (and their applications) onto a slice, follow these steps:
+- Create a slice configuration YAML file (choose the namespaces, clusters, etc to be part of the slice).
+- Use the `kubeslice-cli create sliceConfig` command to apply the slice configuration YAML file.
+
+
+### Creating a Slice
+
+Use the following template to create a slice configuration YAML file.
+:::info
+To understand more about the configuration parameters, see 
+[Slice Configuration Parameters](/versioned_docs/version-0.4.0/reference/configuration-parameters.mdx#slice-configuration-parameters).
+:::
+
+```
+apiVersion: controller.kubeslice.io/v1alpha1
+kind: SliceConfig
+metadata:
+  name: <slice-name>                #The name of the slice
+spec:  
+  sliceSubnet: <slice-subnet>        #The slice subnet
+  sliceType: Application
+  sliceGatewayProvider:
+    sliceGatewayType: OpenVPN
+    sliceCaType: Local
+  sliceIpamType: Local
+  clusters:
+  - <worker-cluster-name1>           #The name of your worker cluster1
+  - <worker-cluster-name2>           #The name of your worker cluster2
+  qosProfileDetails:
+    queueType: HTB
+    priority: 0
+    tcType: BANDWIDTH_CONTROL
+    bandwidthCeilingKbps: 30000
+    bandwidthGuaranteedKbps: 20000
+    dscpClass: AF11
+
+```
+
+### Applying the Slice Configuration YAML file
+
+:::caution
+The `kubeslice-cli create sliceConfig` command returns after the configuration is applied. However, in each cluster, the relevant pods for 
+controlling and managing the slice may still be starting. Please wait for the slice to complete initialization before deploying services to it.
```

Comment:
Can we remove `Please` and start the sentence with `Ensure to wait` or something of that sort...

Reply:
done